master/master.cfg

```diff
@@ -722,31 +722,174 @@ def get_cmake_build_command(builder_type, build_dir, targets=None):
     return cmd
 
 
+# options in LLVM that we always want OFF because they aren't needed
+# for Halide buildbot usage.
+_LLVM_OFF_OPTS = [
+    "CLANG_ENABLE_ARCMT"
+    "CLANG_ENABLE_CLANGD"
+    "CLANG_ENABLE_STATIC_ANALYZER"
+    "CLANG_INCLUDE_DOCS"
+    "CLANG_INSTALL_SCANBUILD"
+    "CLANG_INSTALL_SCANVIEW"
+    "CLANG_PLUGIN_SUPPORT"
+    "CLANG_TIDY_ENABLE_STATIC_ANALYZER"
+    "CLANG_TOOL_APINOTES_TEST_BUILD"
+    "CLANG_TOOL_ARCMT_TEST_BUILD"
+    "CLANG_TOOL_C_ARCMT_TEST_BUILD"
+    "CLANG_TOOL_C_INDEX_TEST_BUILD"
+    "CLANG_TOOL_CLANG_CHECK_BUILD"
+    "CLANG_TOOL_CLANG_DIFF_BUILD"
+    "CLANG_TOOL_CLANG_EXTDEF_MAPPING_BUILD"
+    "CLANG_TOOL_CLANG_FORMAT_BUILD"
+    "CLANG_TOOL_CLANG_FORMAT_VS_BUILD"
+    "CLANG_TOOL_CLANG_FUZZER_BUILD"
+    "CLANG_TOOL_CLANG_IMPORT_TEST_BUILD"
+    "CLANG_TOOL_CLANG_LINKER_WRAPPER_BUILD"
+    "CLANG_TOOL_CLANG_OFFLOAD_BUNDLER_BUILD"
+    "CLANG_TOOL_CLANG_OFFLOAD_PACKAGER_BUILD"
+    "CLANG_TOOL_CLANG_REFACTOR_BUILD"
+    "CLANG_TOOL_CLANG_RENAME_BUILD"
+    "CLANG_TOOL_CLANG_REPL_BUILD"
+    "CLANG_TOOL_CLANG_SCAN_DEPS_BUILD"
+    "CLANG_TOOL_CLANG_SHLIB_BUILD"
+    "CLANG_TOOL_DIAGTOOL_BUILD"
+    "CLANG_TOOL_DICTIONARY_BUILD"
+    "CLANG_TOOL_LIBCLANG_BUILD"
+    "CLANG_TOOL_NVPTX_ARCH_BUILD"
+    "CLANG_TOOL_SCAN_BUILD_BUILD"
+    "CLANG_TOOL_SCAN_BUILD_PY_BUILD"
+    "CLANG_TOOL_SCAN_VIEW_BUILD"
+    "CLANG_TOOLS_EXTRA_INCLUDE_DOCS"
+    "LLVM_BUILD_32_BITS"
```

Comment:
Actually, this shouldn't be here. Either that or the part that conditionally sets it should be removed.